horovod/spark/__init__.py

```diff
@@ -47,28 +48,28 @@ def _task_fn(index, driver_addresses, num_proc, tmout, key):
         else:
             # The rest of tasks need to wait for the first task to finish.
             first_task_addresses = driver_client.all_task_addresses(task_indices_on_this_host[0])
-            first_task_client = task_service.SparkTaskClient(task_indices_on_this_host[0], first_task_addresses, key)
+            first_task_client = task_service.SparkTaskClient(task_indices_on_this_host[0], first_task_addresses, key, settings)
```

Comment:
I'd argue that at the TaskClient level you want to extract verbose flag out of settings, since settings may end up having a lot of information unrelated to TaskClient in future.

Reply:
fixed